specification/digitaltwins/resource-manager/readme.csharp.md

```diff
@@ -1,4 +1,6 @@
-## C
+# C# ADT
```

Comment:
I used these two files as the main inspiration for making the DPS ones, and I fixed a few markdown errors and made them match other files I saw elsewhere while I edited them.